src/Lykke.Job.BlockchainCashinDetector.Core/Domain/CashinStartResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.Job.BlockchainCashinDetector.Core.Domain
+{
+    public enum CashinStartResult
```

Comment:
I suppose, we could combine `CashinResult` and `CashinStartResult` in a single enum by adding all in-progress stuff into `CashinResult`. This looks correct for me 'cause these enums already have intersection in part of `OutdatedBalance`.

Reply:
It's different things, so they expressed in different enums